Description: Exclude transitive ant dependency from cglib because we don't need it
Author: Stuart McCulloch <mcculls@gmail.com>
Bug-Google: https://github.com/google/guice/issues/1174
Last-Update: 2019-07-13

diff --git a/pom.xml b/pom.xml
index ebb0d028..3ee80c56 100644
--- a/pom.xml
+++ b/pom.xml
@@ -160,6 +160,12 @@ See the Apache License Version 2.0 for the specific language governing permissio
         <groupId>cglib</groupId>
         <artifactId>cglib</artifactId>
         <version>3.2.12</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+          </exclusion>
+        </exclusions>
       </dependency>
       <dependency>
         <groupId>junit</groupId>
